### ISSUE TITLE
Cleanup all stuff on resource stop

### ISSUE THREAD

**Issue by Lekenthu** [altmp/altv#260]
The resource-system should delete all created vehicles/checkpoint/voicechannels/colshapes/etc created by it when we stop the resource. We stop the resource because we don't need it, that's why it should delete theese things.

**Comment by FabianTerhorst** [comment]
Yes that's planned already.

**Comment by Lhoerion** [comment]
Duplicate of #193

**Comment by FabianTerhorst** [comment]
no, this isnt a duplicate, this is about serverside created entities.

**Comment by FabianTerhorst** [comment]
Its planned that e.g. vehicles created from a server resource are destroyed when that resource restarts.
